vmdb/spec/models/ems_refresh/refreshers/openstack_refresher_rhos_grizzly_spec.rb

```diff
@@ -67,7 +67,7 @@ def assert_table_counts
     SystemService.count.should       == 0
 
     Relationship.count.should        == 13
-    MiqQueue.count.should            == 19
+    MiqQueue.count.should            == 20
```

Comment:
Sounds great.  Here's the problem: it's been this way for over 2 years (probably much longer).  I can't put this on @Ladas to clean up in this PR.  It would assume @ladas even knows what the original 19 queued messages were and whether or not we should have cared about them.  If we want better testing here, then we need an issue and a new PR for that.

That being said, I can't see how changing the way we reference the event monitor class could add a new message to the queue during refresh.  It's completely unrelated, from what I can tell.

Honestly, I think I'll just ¯\_(ツ)_/¯ and merge.